sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/AvoidExcessiveInheritance.cs

```diff
@@ -103,6 +106,15 @@ protected override void Initialize(ParameterLoadingAnalysisContext context)
                 }, SyntaxKind.ClassDeclaration);

```

Comment:
Not part of this change but this might also apply to interfaces. If you agree, could you create a ticket in the backlog?